.circleci/config.yml

```diff
@@ -220,7 +232,10 @@ workflows:
       - ubuntu_22_04:
           requires:
             - tarball
-      - vscode:
+      - vscode_latest:
+          requires:
+            - tarball
+      - vscode_stable:
```

Comment:
We need someway to not block CI on these

Reply:
ok nevermind, conversation offline indicates that for some reason `release` and `macos` are the only two CI jobs that block so by becoming a dep of `release` this blocks but since latest isn't a dep of `release` or `macos` it doesn't block...I'm not sure where the blocking nature of `release` and `macos` comes from but I'm happy enough with empirical evidence here (which Ashley tested)